libsrc/meshing/meshtype.hpp

```diff
@@ -377,19 +377,19 @@ namespace netgen
 
   public:
     ///
-    Element2d () = default;
-    Element2d (const Element2d &) = default;
-    Element2d (Element2d &&) = default;
-    Element2d & operator= (const Element2d &) = default;
-    Element2d & operator= (Element2d &&) = default;
+    DLL_HEADER Element2d () = default;
+    DLL_HEADER Element2d (const Element2d &) = default;
```

Comment:
DLL_HEADER is not needed for default constructors (Tested with VS2015 on Windows). The others functions are exported with commit 0b411e1b07ddd5dcc0c05fe8a36bca10b9d786d0.